save-preprocessor/src/main/resources/application.properties

```diff
@@ -2,6 +2,7 @@ server.port=5200
 management.endpoints.web.exposure.include=health,info,prometheus
 save.repository=/home/cnb/repositories/
 save.reposFileName=TestSuitesRepos
+save.reposTokenFileName=TestSuitesToken
```

Comment:
Please add an example of how to mount file from host in `docker-compose.override.yaml`

Reply:
Did you mean
```bash
    volumes:
      - '/home/saveu/configs/TestSuitesToken:/workspace/BOOT-INF/classes/TestSuitesToken'
```
?

But we neither added such info for `TestSuitesRepo`, should we do this for `TestSuitesToken` now?

Reply:
Yes, please add this for both

Reply:
Ok, added